### PR TITLE
Rq047 scripts new

### PR DESCRIPTION
**Pull request by maiparkes58**
Can you review these scripts, especially table 4 and 6 (requesting specific clinical codes)? Thanks. M

### CODE REVIEW

File: projects/047 - Lai/template-sql/9-Alcohol-(skin-cohort).template.sql [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	'alcohol-weekly-intake'
+	)
+	AND [Version]=1
+) AND CAST(EventDate AS DATE) < @EndDate;
+
+
+-- The final table========================================================================================================
+SELECT 
+	a.FK_Patient_Link_ID AS PatientId,
+	EventDate,
```
Comment (rw251) [inline projects/047 - Lai/template-sql/9-Alcohol-(skin-cohort).template.sql:98]: "alcohol-weekly-intake" is not the same as the other categories in that you need to check the value (which is the number of units of alcohol per week) in order to put them into one of the categories. Check out the query-patient-alcohol-intake.sql file for an example of what I mean.

**Review by rw251 — COMMENTED** [review]
Just a couple of comments in the files. Also, you don't need to change this, but I find for longitudinal data it's easier to put it all into a single file. The final table would have 4 columns: PatientId, Date, Event, Value. So example rows would be:

832743, 2022-04-02, bmi, 24
832743, 2022-04-02, alcohol-status, heavy-drinker
237472, 2021-12-12, hrt, 

It would mean things like your cohorts would only be defined once and not in every file.  But as I say you don't need to change it - it's just a suggestion for the future.

File: projects/047 - Lai/template-sql/9-Alcohol-(skin-cohort).template.sql (hunk 1 above)
Comment (maiparkes58) [reply projects/047 - Lai/template-sql/9-Alcohol-(skin-cohort).template.sql:96]: Yes "alcohol-weekly-intake" was excluded from the script.

File: projects/047 - Lai/template-sql/2-GPVisit-(both-cohorts).template.sql [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+-- Merge with GP encounter types=================================================================================================================================
+IF OBJECT_ID('tempdb..#GPEncountersFinal') IS NOT NULL DROP TABLE #GPEncountersFinal;
+SELECT FK_Patient_Link_ID, EncounterDate, c.EncounterType
+INTO #GPEncountersFinal
+FROM #GPEncounters g
+LEFT OUTER JOIN #CodingClassifier c ON g.FK_Reference_Coding_ID = c.PK_Reference_Coding_ID
+
+
+-- The final table===============================================================================================================================================
+SELECT DISTINCT FK_Patient_Link_ID AS PatientId, EncounterDate, EncounterType
```
[the remark this answers is not in the document]
Comment (maiparkes58) [reply projects/047 - Lai/template-sql/2-GPVisit-(both-cohorts).template.sql:146]: Yes, noted to the team.

**Review by rw251 — APPROVED** [review]

**Pull request merged**